photonix/photos/models.py

```diff
@@ -232,7 +233,7 @@ def __str__(self):
 
 
 class PhotoTag(UUIDModel, VersionedModel):
-    photo = models.ForeignKey(Photo, related_name='photo_tags', on_delete=models.CASCADE)
+    photo = models.ForeignKey(Photo, related_name='photo_tags', on_delete=models.CASCADE, null=True)
```

Comment:
Is this necessary? Don't we just delete the instance of this model?

Reply:
Yes, It's necessary to add null=True constraint in the photo field Because it's a foreign key field and according to our scenario if a tag assigned to multiple photos and user wants to remove the tag only form a specific photo in that case we can not delete that tag from data base. we have to just remove (disallow) the tag from that particular photo and for that, we have to use remove method. remove method only appear when null=True is set in foreign key field. Please have look in the picture or just go through remove(*objs, bulk=True) section on the URL https://docs.djangoproject.com/en/3.1/ref/models/relations/ .
![Screenshot from 2021-01-25 15-50-17](https://user-images.githubusercontent.com/10600741/105695407-11638f00-5f28-11eb-9780-a00a4cff5347.png)

Reply:
OK, fair enough. Thanks for clarification @GyanP.

Reply:
It's ok!